src/collective/casestudy/profiles/default/registry/main.xml

```diff
@@ -23,18 +25,32 @@
       <element>2.0</element>
     </value>
 
-    <value key="industries">
-      <element>Government</element>
-      <element>Education</element>
-      <element>NGO</element>
-      <element>Corporate</element>
-      <element>Media</element>
+    <value key="industries"
+           purge="false"
+    >
+      <element>gov|Government</element>
+      <element>edu|Education</element>
+      <element>ngo|NGO</element>
+      <element>corporate|Corporate</element>
+      <element>media|Media</element>
     </value>
 
-    <value key="usages">
-      <element>Portal</element>
-      <element>Intranet</element>
-      <element>Knowledge Base</element>
+    <value key="usages"
+           purge="false"
+    >
+      <element>portal|Portal</element>
+      <element>intranet|Intranet</element>
+      <element>kb|Knowledge Base</element>
```

Comment:
Corporate
E-Commerce
Support/Helpdesk
Campaign

Reply:
Let's add those in a new issue